components/automate-chef-io/content/docs/iam-actions.md

```diff
@@ -0,0 +1,36 @@
++++
+title = "IAM Actions"
+description = "Reference Actions for Roles"
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "reference"
+    weight = 60
++++
+
+Reference the chart on this page when creating a *Role* to know which action grants access to what page in the browser.
+
+*IAM Action* lists the associated action or actions required to access that page in the browser.
```

Comment:
we may want to have a note that the * in these actions will give get, list, delete, create, etc permissions, and they can be more specific if they want to restrict that